MonoGame.Framework/Utilities/OS.cs

```diff
@@ -0,0 +1,46 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+namespace MonoGame.Utilities
+{
+    /// <summary>
+    /// Defines the operating system.
+    /// </summary>
+    public enum OS
```

Comment:
> You can only really do cosmetic things like described above.

Setting default storage folder? Keyboard input? On Windows and Linux Ctrl + C = Copy, but on Mac it's different, it's Command + C. Your point is pointless... sorry I couldn't resist saying it...